src/js/auth.js

```diff
@@ -33,6 +34,12 @@ function getAllowedUnauthedPaths() {
 }
 
 export function allowUnauthed() {
+  /**
+   * We can delete the whole logic after console.dot migration is over
+   */
+  if (getEnv() !== 'prod') {
```

Comment:
I would check here for the `console` in URL. If that is present we will return `false` all the time. I would presonally add new helper function exposed on window to check this as well.

Reply:
Yeah, it struck my mind too but we basically have to stop the chrome code promotion again until the console switch happens. I don't think we want to that ATM.